aten/src/ATen/core/jit_type.h

```diff
@@ -548,7 +555,7 @@ struct VaryingShape {
     return dims_ == other.dims_;
   }
 
-  const c10::optional<T>& operator[](int i) const {
+  const c10::optional<T> &operator[](size_t i) const {
```

Comment:
Nit: this seems to include unnecessary formatting change which I assume would break clang-format?

Reply:
this **was** clang-format :-) But let's see if it breaks our lint jobs.